js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -16,4 +32,42 @@ describe("OperatorSignInModal", () => {
 
     expect(view.queryByText(/fit for duty check/i)).not.toBeInTheDocument();
   });
+
+  test("submits successful attestation to the server", async () => {
+    putMetaData("csrf-token", "TEST-CSRF-TOKEN");
+    (fetch as jest.Mock).mockResolvedValueOnce({
```

Comment:
**suggestion:** I believe this can be rewritten as:
```suggestion
    jest.mocked(fetch).mockResolvedValueOnce({
```
which provides a bit more type safety.

Reply:
thanks! f1b55ed